src/main/webapp/app/modules/canvas/filter/filter-panel.scss

```diff
@@ -0,0 +1,37 @@
+.FilterPanel-Main {
+  .FilterPanel {
```

Comment:
can you use view component and add all the props there as hard coding color may lead an issue